app/labor/daily_email_logic.rb

```diff
@@ -4,18 +4,21 @@ def initialize(user)
   end
 
   def get_article_to_send
-    articles_with_user_tags.
-      where.not(user: @user).
-      order(hotness_score: :desc).
-      first
+    if get_user_tag_names != []
```

Comment:
I like the way this handles the possibility that a user may not have any tags followed yet and still sends a relevant email.

Reply:
Thanks for adding this in.